test/EbUnitTest.h

```diff
@@ -23,4 +23,18 @@ extern const char eb_unit_test_result_str[2][25];
 }
 #endif
 
-#endif  // EbUnitTest_h
+#ifdef _MSC_VER
```

Comment:
```suggestion
#ifdef _WIN32
```

MinGW-w64 uses `_aligned_malloc` too and lacks `posix_memalign` generally